release/readme.html

```diff
@@ -205,119 +254,192 @@ <h2><a id="Weather_Data_63"></a>Weather Data</h2>
+        <dd>
+            Another high impact change with this release is the ability to leverage GPU calculations to improve runtime
+            performance of the simulation.  The documentation contains plenty of information to let users take
+            advantage of this new feature, which has the potential to drastically speed up simulation runtime, however,
+            a few notes provide high level tips for this new capability:
+            <ul class="list-group">
+                <li class="list-group-item">
+                    There is a performance overhead associated with passing calculations between the GPU and the CPU, with the
+                    GPU becoming noticeably more efficient in models with more than around 200 shadow-casting surfaces. The
+                    actual number where the GPU becomes more efficient will depend on the specs of the respective hardware.
+                </li>
+                <li class="list-group-item">
+                    Some machines (mostly server machines) do not have GPU hardware or the required graphics drivers to utilize
+                    this capability. If this scenario is detected on a computer, EnergyPlus will issue a warning and proceed
+                    with the simulation using the calculations on the CPU.
+                </li>
+                <li class="list-group-item">
+                    Some computers have multiple GPUs. In this case, the highest performance GPU is not always used by default.
+                    You may want to select which GPU is used when running EnergyPlus by setting the graphics performance
+                    preferences on your computer.
+                </li>
+            </ul>
+        </dd>
+    </dl>
+
+    <h2>Highlights of this release (V9.3.0)</h2>
+    <h3><a id="Highlights_of_this_release"></a>New Features</h3>
     <table class="table table-striped table-bordered">
         <tr>
             <th>PR #</th>
             <th>Description</th>
         </tr>
         <tr>
-            <td><a href="https://github.com/NREL/EnergyPlus/pull/6749">6749</a></td>
-            <td>Refactor Table Objects and Interpolation Algorithms</td>
+            <td><a href="https://github.com/NREL/EnergyPlus/pull/7489">7489</a></td>
+            <td>Extend EIR WWHP to Air-source</td>
         </tr>
         <tr>
-            <td><a href="https://github.com/NREL/EnergyPlus/pull/7217">7217</a></td>
-            <td>Increases Schedule:Compact IDD fields</td>
+            <td><a href="https://github.com/NREL/EnergyPlus/pull/7562">7562</a></td>
+            <td>Add Three Alternative Models for Clear Sky Emissivity Calculation</td>
         </tr>
         <tr>
-            <td><a href="https://github.com/NREL/EnergyPlus/pull/7230">7230</a></td>
-            <td>Model DOAS Supplying Air to Inlets of Multiple AHUs</td>
+            <td><a href="https://github.com/NREL/EnergyPlus/pull/7603">7603</a></td>
+            <td>Input file conversion utility</td>
```

Comment:
I changed the title of this PR to "Input file conversion utility IDF->epJSON and reverse".

Reply:
Updated in this section.